src/main/kotlin/com/ampnet/payoutservice/config/ApplicationProperties.kt

```diff
@@ -10,6 +10,7 @@ class ApplicationProperties {
     val jwt = JwtProperties()
     val ipfs = IpfsProperties()
     val payout = PayoutProperties()
+    val createPayoutQueue = CreatePayoutQueueProperties()
```

Comment:
I would actually leave naming here as-is since this is specific to create payout tasks, and if we add more queues in the future they would have different names.